src/Van/Handlers/VanDisplayHandlerV2.h

```diff
@@ -214,7 +214,7 @@ class VanDisplayHandlerV2 : public AbstractVanMessageHandler {
             leftStickButtonReturn = currentTime + LEFT_STICK_BUTTON_TIME;
             ignitionDataToBridge.LeftStickButtonPressed = 1;
             dataToBridge.LeftStickButtonPressed = 1;
-            //canTripInfoHandler->TripButtonPress();
+            canTripInfoHandler->TripButtonPress();
```

Comment:
Is this needed for the new popup handler functionality?

Reply:
No, I didn't notice this line.